File: TOMIMSV4/Entity/Mantenimientos/Empresa_Transporte_Vehiculo/clsBeEmpresa_transporte_vehiculos.vb
Public Class clsBeEmpresa_transporte_vehiculos
    Implements ICloneable

    Public Property IdVehiculo() As Integer = 0
    Public Property IdEmpresaTransporte() As Integer = 0
    Public Property IdTipoContenedor() As Integer = 0
    Public Property Placa() As String = ""
    Public Property Marca() As String = ""
    Public Property Modelo() As String = ""
    Public Property Peso() As Double = 0.0
    Public Property Volumen() As Double = 0.0
    Public Property Activo() As Boolean = False
    Public Property User_agr() As String = ""
    Public Property Fec_agr() As Date = Date.Now
    Public Property User_mod() As String = ""
    Public Property Fec_mod() As Date = Date.Now
    Public Property Tipo() As String = ""
    Public Property Alto() As Double = 0.0
    Public Property Largo() As Double = 0.0
    Public Property Ancho() As Double = 0.0
    Public Property Placa_comercial() As String = ""
    Public Property Es_contedor() As Integer = 0

    Sub New()
    End Sub

    Sub New(ByRef IdVehiculo As Integer, ByVal IdEmpresaTransporte As Integer, ByVal IdTipoContenedor As Integer, ByVal placa As String, ByVal marca As String, ByVal modelo As String, ByVal peso As Double, ByVal volumen As Double, ByVal activo As Boolean, ByVal user_agr As String, ByVal fec_agr As Date, ByVal user_mod As String, ByVal fec_mod As Date, ByVal tipo As String, ByVal alto As Double, ByVal largo As Double, ByVal ancho As Double, ByVal placa_comercial As String, ByVal es_contedor As Integer)
        Me.IdVehiculo = IdVehiculo
        Me.IdEmpresaTransporte = IdEmpresaTransporte
        Me.IdTipoContenedor = IdTipoContenedor
        Me.Placa = placa
        Me.Marca = marca
        Me.Modelo = modelo
        Me.Peso = peso
        Me.Volumen = volumen
        Me.Activo = activo
        Me.User_agr = user_agr
        Me.Fec_agr = fec_agr
        Me.User_mod = user_mod
        Me.Fec_mod = fec_mod
        Me.Tipo = tipo
        Me.Alto = alto
        Me.Largo = largo
        Me.Ancho = ancho
        Me.Placa_comercial = placa_comercial
        Me.Es_contedor = es_contedor
    End Sub

    Public Function Clone() As Object Implements System.ICloneable.Clone
        Return MyBase.MemberwiseClone()
    End Function

End Class

File: WMS.EntityCore/Datos_Maestros/clsBeEmpresa_transporte_vehiculos.cs
using System.ComponentModel;
using System.ComponentModel.DataAnnotations.Schema;

namespace WMSWebAPI.Be
{
    public class clsBeEmpresa_transporte_vehiculos : ICloneable
    {
        [Column("IdVehiculo")]
        [DisplayName("IdVehiculo")]
        public int IdVehiculo { get; set; } = 0;

        [Column("IdEmpresaTransporte")]
        [DisplayName("IdEmpresaTransporte")]
        public int IdEmpresaTransporte { get; set; } = 0;

        [Column("IdTipoContenedor")]
        [DisplayName("IdTipoContenedor")]
        public int IdTipoContenedor { get; set; } = 0;

        [Column("placa")]
        [DisplayName("placa")]
        public string Placa { get; set; } = "";

        [Column("marca")]
        [DisplayName("marca")]
        public string Marca { get; set; } = "";

        [Column("modelo")]
        [DisplayName("modelo")]
        public string Modelo { get; set; } = "";

        [Column("peso")]
        [DisplayName("peso")]
        public double Peso { get; set; } = 0;

        [Column("volumen")]
        [DisplayName("volumen")]
        public double Volumen { get; set; } = 0;

        [Column("activo")]
        [DisplayName("activo")]
        public bool Activo { get; set; } = false;

        [Column("user_agr")]
        [DisplayName("user_agr")]
        public string User_agr { get; set; } = "";

        [Column("fec_agr")]
        [DisplayName("fec_agr")]
        public DateTime Fec_agr { get; set; } = DateTime.Now;

        [Column("user_mod")]
        [DisplayName("user_mod")]
        public string User_mod { get; set; } = "";

        [Column("fec_mod")]
        [DisplayName("fec_mod")]
        public DateTime Fec_mod { get; set; } = DateTime.Now;

        [Column("tipo")]
        [DisplayName("tipo")]
        public string Tipo { get; set; } = "";

        [Column("alto")]
        [DisplayName("alto")]
        public double Alto { get; set; } = 0;

        [Column("largo")]
        [DisplayName("largo")]
        public double Largo { get; set; } = 0;

        [Column("ancho")]
        [DisplayName("ancho")]
        public double Ancho { get; set; } = 0;

        [Column("placa_comercial")]
        [DisplayName("placa_comercial")]
        public string Placa_comercial { get; set; } = "";

        [Column("es_contedor")]
        [DisplayName("es_contedor")]
        public int Es_contedor { get; set; } = 0;

        public clsBeEmpresa_transporte_vehiculos() { }

        public object Clone()
        {
            return this.MemberwiseClone();
        }
    }
}
